### ISSUE TITLE
Regarding training loss

### ISSUE THREAD

**Issue by laukun** [xiaomengyc/ACoL#4]
Thank author for the code.
 From the figures showing CAM and yours, there's not much difference on the mined extent of the object. Do you have the loss curves of training on some dataset before and after adding ACoL?

**Comment by xiaomengyc** [comment]
@laukun Sorry, I did not compare the loss curves between the two methods.  Although these two methods have the same mathematical formalization, the network structures are not exactly the same. Also, the training process will be influenced by the stochastic optimization process. So, I think there will be some differences between the two convergent curves.
